runtime/CMakeLists.txt

```diff
@@ -75,8 +75,10 @@ include(cmake/J9vmGenAsm.cmake)
 # to explicitly tag individual targets which seems tedious.
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-if(OMR_HOST_OS STREQUAL "osx")
+if(OMR_OS_OSX)
 	set(CMAKE_INSTALL_RPATH "@loader_path")
+elseif(OMR_OS_AIX)
+	set(CMAKE_INSTALL_RPATH ".")
```

Comment:
Where is the equivalent to this in the non-cmake world? I don't find a rpath setting for AIX.